services/base-static.js

```diff
@@ -4,53 +4,49 @@ const makeBadge = require('../gh-badges/lib/make-badge')
 const { makeSend } = require('../lib/result-sender')
 const analytics = require('../lib/analytics')
 const BaseService = require('./base')
-
-const serverStartTime = new Date(new Date().toGMTString())
+const {
+  serverHasBeenUpSinceResourceCached,
+  setCacheHeadersForStaticResource,
+} = require('./cache-headers')
 
 module.exports = class BaseStaticService extends BaseService {
```

Comment:
There are no changes in behavior here. It's a straight refactor of functionality.